Sprinkle of Yeezus/Settings.swift

```diff
@@ -12,67 +12,185 @@ import UserNotifications
     
+    
     private func sprinkleNotifications(_ sprinkleList: Array<Sprinkle>) {
         notificationCenter.removeAllPendingNotificationRequests()
         
-        guard Switch.isOn else {
+        guard notificationSwitch.isOn else { //Runs only user turns on notification switch
             return
         }
         
         let notification = UNMutableNotificationContent()
-        let notificationText = pickRandomQuote(sprinkleList)
-        notification.body = "\"\(notificationText.quote)\" \(notificationText.quoteSource), \(notificationText.date)"
-
-        let components = Calendar.current.dateComponents([.hour, .minute], from: sprinkleTimePicked)
-        let hour = components.hour ?? 12
-        let minute = components.minute ?? 0
+        var notificationText = [Sprinkle]()
+        var components = Calendar.current.dateComponents([.hour, .minute], from: sprinkleTimePicked) //grabs time from picker
+        
+        let today = Date()
+        let calendar = Calendar.current
+        components.day = calendar.component(.day, from: today) //grabs current day from phones calendar
+        let month = 30
+        var nextDay = DateComponents()
+        nextDay.hour = components.hour
+        nextDay.minute = components.minute
+        
+        
+        //schedules sprinkles for 30 days
+       for days in 0...month {
+        notificationText.append(pickRandomQuote(sprinkleList))
         
-        InfoLabel.text = "Sprinkles will be sent every day at \(hour):\(minute)"
+        notification.body = "\"\(notificationText[days].quote)\"" //Formats notification
+        if notificationText[days].quoteSource != "" {
+            notification.body.append(" - \(notificationText[days].quoteSource)")
+        }
+        if notificationText[days].date != 0 {
+            notification.body.append(", \(notificationText[days].date)")
+        }
         
-        let trigger = UNCalendarNotificationTrigger(dateMatching: components, repeats: true)
-        let request = UNNotificationRequest(identifier: "Sprinkle-of-Yeezus", content: notification, trigger: trigger)
+        nextDay.day = components.day! + days //tommorow is today plus 1 and the time set
+        let trigger = UNCalendarNotificationTrigger(dateMatching: nextDay, repeats: false)
+        let request = UNNotificationRequest(identifier: "Sprinkle number \(days)", content: notification, trigger: trigger)
+        print("Time set for \(nextDay)")
+        print("Notification content: \(notificationText[days].quote)")
         
         notificationCenter.add(request) { error in
             if let error = error {
                 print(error)
+                }
             }
         }
+        updateLabelTime()
+    }
+    
+    private func anniversaryNotifications() {
+        guard notificationSwitch.isOn else { //Runs only user turns on notification switch
+            return
+        }
+        
+        let today = Date()
+        let currentCalendar = Calendar.current
```

Comment:
You are using `Calendar.current` a lot. Maybe make it a `private let calendar = Calendar.current` in the class